app/Livewire/Project/Service/Storage.php

```
@@ -14,6 +14,22 @@ class Storage extends Component
+
+    public string $name = '';
+
+    public string $mount_path = '';
+
+    public ?string $host_path = null;
+
+    public string $file_storage_path = '';
+
+    public ?string $file_storage_content = null;
+
+    public string $file_storage_directory_source = '';
```

PHPMD (2.15.0)

Avoid excessively long variable names like $file_storage_directory_source. Keep variable name length under 20. (undefined)

(LongVariable)